nixio/multi_tag.py

```diff
@@ -74,7 +82,16 @@ def extents(self, da):
         if da is None:
             del self._h5group["extents"]
         else:
-            self._h5group.create_link(da, "extents")
+            ext = da
+            if not isinstance(da, DataArray):
+                blk = self._parent
+                name = "{}-extents".format(self.name)
+                if name in blk.data_arrays:
+                    del blk.data_arrays[name]
+                ext = blk.create_data_array(name,
+                                            "{}-extents".format(self.type)
+                                            , data=da)
+            self._h5group.create_link(ext, "extents")
```

Comment:
Given my comments above about not having the convenience functionality in the `positions` setter, I think the same should apply to `extents`.
I'm thinking we could have an `extents` argument in the `create_multi_tag()` method that works the same way as `positions`.

In both cases (for positions and extents), I think it only really makes sense to allow the user to automatically create DataArrays for MultiTag `positons` and `extents` **once** per MultiTag, on creation.  Otherwise, we're creating an incentive for users to create, recreate, and change the positions and extents through these creation functions and that would lead to losing track of DataArray creation, causing naming conflicts and other issues.